runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/BenefitsSection/BenefitsCarusel/BenefitsCarusel.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC } from 'react';
+
+import Image from 'next/image';
+
+import { translate } from '#src-app/hooks/useTranslations';
+import CaruselButton from '#src-landing/components/CaruselButton';
+import Typography from '#src-landing/components/Typography';
+
+import styles from './BenefitsCarusel.module.scss';
+import { Slide } from './BenefitsCarusel.types';
+import MobileCaruselNav from './MobileCaruselNav';
+
+const SLIDES: Slide[] = [
+    {
+        descriptionKey: 'Landing.Benefits.Slides.Repetitive.Description',
+        imageSrc: '/images/shapes/beach_access.svg',
+        imageAlt: 'beach icon',
+        titleKey: 'Landing.Benefits.Slides.Repetitive.Title',
+    },
+    {
+        descriptionKey: 'Landing.Benefits.Slides.Efficiency.Description',        
+        imageSrc: '/images/shapes/hourglass.svg',
+        imageAlt: 'Hourglass icon',
+        titleKey: 'Landing.Benefits.Slides.Efficiency.Title',
+    },
+    {
+        descriptionKey: 'Landing.Benefits.Slides.Accuracy.Description',        
+        imageSrc: '/images/shapes/layers.svg',
+        imageAlt: 'Layers icon',
+        titleKey: 'Landing.Benefits.Slides.Accuracy.Title',
+    }
+];
+
+const BenefitsCarusel: FC = () => {
+    const [currentSlide, setCurrentSlide] = React.useState(0);
+
+    const incrementSlide = () => {
+        if (currentSlide === SLIDES.length - 1) {
+            setCurrentSlide(0);
+        } else {
+            setCurrentSlide(currentSlide + 1);
+        }
+    };
+
+    const decrementSlide = () => {
+        if (currentSlide === 0) {
+            setCurrentSlide(SLIDES.length - 1);
+        } else {
+            setCurrentSlide(currentSlide - 1);
+        }
+    };
+    
+
+    return (
+        <div className={styles.root}>
```

Comment:
shouldn't it be smth like `section`?

Reply:
semantic html ftw

Reply:
I see that there is a section in the parent component, lets see if we can do smth different here

Reply:
we ran out of semantic tags... i mean, we can wrap content inside `article`?

Reply:
for sure we can use article to cover carousel elements `An article should make sense on its own and it should be possible to distribute it independently from the rest of the site.

`